uptycs/manifest.json

```diff
@@ -0,0 +1,40 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "d27ee4b6-649d-42bd-b7ac-fb40537d7031",
+  "app_id": "uptycs",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Collect alerts, detections data from Uptycs platform",
+    "title": "Uptycs",
+    "media": [
+     {
+        "media_type": "image",
+        "caption": "Uptycs logo",
+        "image_url": "logos/uptycs-logo.png"
```

Comment:
The logo will automatically be placed on your tile, and does not need to be added here. Instead, we recommend adding screenshots of your dashboard, and of your platform, to give users a good idea of what to expect from this integration.